src/aggregate/aggregate_exec.c

```diff
@@ -100,7 +100,9 @@ static size_t serializeResult(AREQ *req, RedisModuleCtx *outctx, const SearchRes
     count++;
     RLookup *lk = AGPLN_GetLookup(&req->ap, NULL, AGPLN_GETLOOKUP_LAST);
 
-    size_t nfields = RLookup_GetLength(lk, &r->rowdata, req->outFields.explicitReturn);
+    int excludeFlags =
```

Comment:
excludeFlags is F_HIDDEN, includeFlags is either 0 or F_EXPLICITRETURN

Reply:
My bad